### ISSUE TITLE
Google Chrome тормозит при включенной синхронизации когда расширение активно

### ISSUE THREAD

**Issue by popov654** [anticensority/runet-censorship-bypass#246]
Происходит это при открытии новой вкладки через кнопку на панели вкладок, при сохранении страницы в закладки и удалении из закладок, и возможно при некоторых других действиях. Меня особенно раздражает момент с сохранением в закладки, поскольку я пользуюсь этой функцией часто. Синхронизацию журнала и открытых вкладок я отключил, т.к. мне это не нужно.

Добавление домена google.com в список непроксируемых не помогает. Что можно сделать, чтобы исправить ситуацию? Возможно, я добавляю не тот домен?

**Comment by ilyaigpetrov** [comment]
Без синхронизации тоже тормозит?
Попробуйте протестировать на только что созданном профиле Хрома с установленным расширением и без него.
Или в Хромиуме: там нет синхронизации.

**Comment by popov654** [comment]
Ничего не понимаю. Выключил синхронизацию, а затем снова включил - при добавлении и удалении закладки тормозить перестало. Но тормозит, например, если пару раз нажать Ctrl+Shift+T чтобы вернуть закрытую недавно вкладку. Открытие новой вкладки тоже всё ещё тормозит.

На самом деле, возможно проблема в моём SSD, на котором стоит система, с ним давно происходят очень странные тормоза. Но именно сёрфинг в хром у меня никогда раньше не лагал...

Это может быть связано с тем, что ваш PAC скрипт стал тяжелее, и мой процессор его банально "не вывозит"? У меня Xeon E-5 2690v2, и у него довольно низкая тактовая частота по современным меркам.

Но что характерно, если включить ваше расширение, тормоза реально пропадают. Также если тормозит открытие конкретной страницы, и сайт при этом в РФ не блокируется, то после добавления его в белый список загрузка тоже перестаёт тормозить. Там правда тормоза совсем небольшие, но всё же ощущается как "статтеры", неприятно.

**Comment by popov654** [comment]
Ничего не понимаю. Выключил синхронизацию, а затем снова включил - при добавлении и удалении закладки тормозить перестало. Но тормозит, например, если пару раз нажать Ctrl+Shift+T чтобы вернуть закрытую недавно вкладку. Открытие новой вкладки тоже всё ещё тормозит.

На самом деле, возможно проблема в моём SSD, на котором стоит система, с ним давно происходят очень странные тормоза. Но именно сёрфинг в хром у меня никогда раньше не лагал...

Это может быть связано с тем, что ваш PAC скрипт стал тяжелее, и мой процессор его банально "не вывозит"? У меня Xeon E-5 2690v2, и у него довольно низкая тактовая частота по современным меркам.

Но что характерно, если выключить ваше расширение, тормоза реально пропадают. Также если тормозит открытие конкретной страницы, и сайт при этом в РФ не блокируется, то после добавления его в белый список загрузка тоже перестаёт тормозить. Там правда тормоза совсем небольшие, но всё же ощущается как "статтеры", неприятно.

**Comment by popov654** [comment]
Ничего не понимаю. Выключил синхронизацию, а затем снова включил - при добавлении и удалении закладки тормозить перестало. Но тормозит, например, если пару раз нажать Ctrl+Shift+T чтобы вернуть закрытую недавно вкладку. Открытие новой вкладки тоже всё ещё тормозит.

На самом деле, возможно проблема в моём SSD, на котором стоит система, с ним давно происходят очень странные тормоза. Но именно сёрфинг в хром у меня никогда раньше не лагал...

Это может быть связано с тем, что ваш PAC скрипт стал тяжелее, и мой процессор его банально "не вывозит"? У меня Xeon E5-2690v2, и у него довольно низкая тактовая частота по современным меркам.

Но что характерно, если выключить ваше расширение, тормоза реально пропадают. Также если тормозит открытие конкретной страницы, и сайт при этом в РФ не блокируется, то после добавления его в белый список загрузка тоже перестаёт тормозить. Там правда тормоза совсем небольшие, но всё же ощущается как "статтеры", неприятно.

**Comment by popov654** [comment]
> Попробуйте протестировать на только что созданном профиле Хрома

Этот профиль относительно свежий, ему всего месяц, даже меньше. Всё потому, что я случайно потерял свой старый профиль, когда пытался откатить Хром на версию 121 до богомерзкого изменения UI строки вкладок.

**Comment by popov654** [comment]
Нет, поправка - при добавлении в закладки всё так же тормозит, просто чуть поменьше.

**Comment by ilyaigpetrov** [comment]
У вас там выше три примерно одинаковых сообщения опубликовалось -- удалите лишние.

**Comment by popov654** [comment]
Я удалял, и у меня они не отображаются почему-то... Попробуйте обновить страницу?

**Comment by popov654** [comment]
Кажется, помогло добавление в белый список домена googleapis.com

**Comment by popov654** [comment]
Не помогло. Так же тормозит, но не постоянно, один раз из двух или трёх примерно

**Comment by ilyaigpetrov** [comment]
Всё же попробуйте создать чистый профиль Хрома, но не привязывать его к аккаунту Google.
Протестируйте без расширения, потом с ним.

**Comment by Verity-Freedom** [comment]
Учитывая что вы что-то там пытались откатывать и ssd у вас умирает на расширение грешить не стоит...

**Comment by popov654** [comment]
> Учитывая что вы что-то там пытались откатывать и ssd у вас умирает на расширение грешить не стоит...

Извините, но это довольно спорно... Дело в том, что у меня друг живёт за границей, и он мне дал доступ к своему личному VPN. Так вот, парадоксально, но через его VPN у меня сайты открываются быстрее, чем напрямую. Мне кажется, лаги могут быть из-за моего провайдера, который что-то там проверяет у себя на лету.

А SSD - не умирает, его состояние отличное, если верить S.M.A.R.T. Он просто вот такой вот хреновый оказался.
